src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -3654,6 +3670,20 @@ public static Value to_url(ScriptRuntime controller, final Value value) {
         : new Value(adventure.getRequest().getURLString());
   }
 
+  public static Value to_wiki_url(ScriptRuntime controller, final Value value) {
+    var type =
+        switch (value.getType().getName()) {
```

Comment:
can you not switch on `getType()` directly?

Reply:
No, switches have to be primitives or enums!

Reply:
And before you go in and try to enum types... the issue is that we use the same class to make arbitrary types at runtime

Reply:
I thought `getType` was a TypeSpec.

Reply:
okay, that's `value.getType().getType()`. Can you switch on that instead for slightly less stringly-typed code?

Reply:
yeah ok